From 4ed40e67197778b28e4b822558614a517242e3e1 Mon Sep 17 00:00:00 2001
From: Marian Csontos <mcsontos@redhat.com>
Date: Thu, 3 Oct 2024 16:14:11 +0200
Subject: [PATCH 01/13] RHEL9

---
 VERSION    | 2 +-
 VERSION_DM | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/VERSION b/VERSION
index 7619e491b..f96b435a3 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-2.03.28(2) (2024-11-04)
+2.03.28(2)-RHEL9 (2024-11-04)
diff --git a/VERSION_DM b/VERSION_DM
index 16d64d69d..cd0c694ae 100644
--- a/VERSION_DM
+++ b/VERSION_DM
@@ -1 +1 @@
-1.02.202 (2024-11-04)
+1.02.202-RHEL9 (2024-11-04)
-- 
2.47.0

