From a5a2d5fa1ec47a5a548db4cf435dc84de7ce7c31 Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Wed, 20 Oct 2021 16:12:41 -0500
Subject: [PATCH 03/54] pvscan: fix messages from coverity changes

---
 tools/pvscan.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/pvscan.c b/tools/pvscan.c
index 8e2611361..f60c4a2ca 100644
--- a/tools/pvscan.c
+++ b/tools/pvscan.c
@@ -1354,7 +1354,7 @@ static int _online_devs(struct cmd_context *cmd, int do_all, struct dm_list *pvs
 		devsize = dev->size;
 		if (!devsize &&
 		    !dev_get_size(dev, &devsize)) {
-			log_print("pvscan[%d] PV %s can get device size.", getpid(), dev_name(dev));
+			log_print_pvscan(cmd, "PV %s missing device size.", dev_name(dev));
 			release_vg(vg);
 			continue;
 		}
@@ -1786,7 +1786,7 @@ static int _pvscan_cache_args(struct cmd_context *cmd, int argc, char **argv,
 		int has_pvid;
 
 		if (!label_read_pvid(devl->dev, &has_pvid)) {
-			log_print("pvscan[%d] %s cannot read.", getpid(), dev_name(devl->dev));
+			log_print_pvscan(cmd, "%s cannot read label.", dev_name(devl->dev));
 			dm_list_del(&devl->list);
 			continue;
 		}
-- 
2.34.3

