From e6f8d9855336dbf1cbf212f60b84a03946365795 Mon Sep 17 00:00:00 2001
From: Marian Csontos <mcsontos@redhat.com>
Date: Thu, 16 May 2024 15:34:28 +0200
Subject: [PATCH 07/13] WHATS_NEW: update

---
 WHATS_NEW_DM | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/WHATS_NEW_DM b/WHATS_NEW_DM
index f8ca946d6..74b356bf2 100644
--- a/WHATS_NEW_DM
+++ b/WHATS_NEW_DM
@@ -1,3 +1,10 @@
+Version 1.02.203 - 
+=====================================
+  Revert Increase DM_UDEV_RULES_VSN to 3 to indicate changed udev rules.
+  Revert Rename DM_NOSCAN to .DM_NOSCAN so it's not stored in udev db.
+  Revert Rename DM_SUSPENDED to .DM_SUSPENDED so it's not stored in udev db.
+  Revert Do not import DM_UDEV_DISABLE_OTHER_RULES_FLAG from db in 10-dm-disk.rules.
+
 Version 1.02.202 - 04th November 2024
 =====================================
   Introduce dm_config_parse_only_section to stop parsing after section.
-- 
2.47.0

