From a468c4028a2a996e5651feaf728d4c126698d478 Mon Sep 17 00:00:00 2001
From: Marian Csontos <mcsontos@redhat.com>
Date: Thu, 9 Jan 2025 15:59:48 +0100
Subject: [PATCH 27/27] WHATS_NEW: update

---
 WHATS_NEW | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WHATS_NEW b/WHATS_NEW
index 40c6c2cd0..07493831f 100644
--- a/WHATS_NEW
+++ b/WHATS_NEW
@@ -1,5 +1,6 @@
 Version 2.03.30 - 
 ==================
+  Fix support for disabling memory locking (2.03.27).
   Do not extend an LV if FS resize unsupported and '--fs resize' used.
   lvconvert detects early volume in use when converting it to a pool.
 
-- 
2.47.1

