From b78f20df30d214ab4d7e7f5846e0c3da8c0938ab Mon Sep 17 00:00:00 2001
From: Marian Csontos <mcsontos@redhat.com>
Date: Wed, 7 Jan 2026 11:27:26 +0100
Subject: [PATCH 29/32] autoreconf: reconfigure

---
 configure              | 20 ++++++++++++++++++++
 include/configure.h.in |  3 +++
 2 files changed, 23 insertions(+)

diff --git a/configure b/configure
index 935b1b74a..2371dce55 100755
--- a/configure
+++ b/configure
@@ -6921,6 +6921,20 @@ then :
 
 fi
 
+       for ac_header in xfs/xfs.h
+do :
+  ac_fn_c_check_header_compile "$LINENO" "xfs/xfs.h" "ac_cv_header_xfs_xfs_h" "#define _GNU_SOURCE 1
+"
+if test "x$ac_cv_header_xfs_xfs_h" = xyes
+then :
+  printf "%s\n" "#define HAVE_XFS_XFS_H 1" >>confdefs.h
+ LVM_NO_XFS_WARN=
+else case e in #(
+  e) LVM_NO_XFS_WARN=y ;;
+esac
+fi
+
+done
        for ac_header in libaio.h
 do :
   ac_fn_c_check_header_compile "$LINENO" "libaio.h" "ac_cv_header_libaio_h" "$ac_includes_default"
@@ -18237,6 +18251,12 @@ then :
 printf "%s\n" "$as_me: WARNING: Only libdm part can be build without libaio:  make [install_]device-mapper" >&2;}
 fi
 
+if test -n "$LVM_NO_XFS_WARN"
+then :
+  { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: 'lvextend --resizefs' is less capable without xfs/xfs.h (see package: xfsprogs devel)" >&5
+printf "%s\n" "$as_me: WARNING: 'lvextend --resizefs' is less capable without xfs/xfs.h (see package: xfsprogs devel)" >&2;}
+fi
+
 if test "$ODIRECT" != "yes"
 then :
   { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: O_DIRECT disabled: low-memory pvmove may lock up" >&5
diff --git a/include/configure.h.in b/include/configure.h.in
index 51352c090..8ea936f18 100644
--- a/include/configure.h.in
+++ b/include/configure.h.in
@@ -549,6 +549,9 @@
 /* Define to 1 if 'vfork' works. */
 #undef HAVE_WORKING_VFORK
 
+/* Define to 1 if you have the <xfs/xfs.h> header file. */
+#undef HAVE_XFS_XFS_H
+
 /* Define to 1 if the system has the type '_Bool'. */
 #undef HAVE__BOOL
 
-- 
2.52.0

