From e027f4da9bc7b4ed9b225af75089e3443595bf81 Mon Sep 17 00:00:00 2001
From: David Teigland <teigland@redhat.com>
Date: Mon, 2 May 2022 09:46:28 -0500
Subject: [PATCH 46/54] improve description of devices option

---
 tools/args.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tools/args.h b/tools/args.h
index 9a7bf81b2..00a2ec817 100644
--- a/tools/args.h
+++ b/tools/args.h
@@ -235,8 +235,9 @@ arg(deviceidtype_ARG, '\0', "deviceidtype", string_VAL, 0, 0,
     "then it will override the default type that lvm would use.\n")
 
 arg(devices_ARG, '\0', "devices", pv_VAL, ARG_GROUPABLE, 0,
-    "Devices that the command can use. This option can be repeated\n"
-    "or accepts a comma separated list of devices. This overrides\n"
+    "Restricts the devices that are visible and accessible to the command.\n"
+    "Devices not listed will appear to be missing. This option can be\n"
+    "repeated, or accepts a comma separated list of devices. This overrides\n"
     "the devices file.\n")
 
 arg(devicesfile_ARG, '\0', "devicesfile", string_VAL, 0, 0,
-- 
2.34.3

